gui/default/syncthing/core/syncthingController.js

```diff
@@ -1509,13 +1495,18 @@ angular.module('syncthing.core')
                 delete folderCfg.versioning;
             }
 
+            if (!$scope.editingExisting && $('#editIgnores textarea').val()) {
```

Comment:
You should store the return value in a variable, as the $() lookup is quite expensive.